yksgi/src/lib.rs

```diff
@@ -30,6 +30,9 @@ static ALWAYS_SKIP_FUNCS: [&str; 8] = [
 static SKIP_FOR_NOW_FUNCS: [&str; 2] = ["yk_location_drop", "yk_mt_drop"];
 static YK_CONTROL_POINT_FUNC: &str = "__ykrt_control_point";
 
+static POINTER_USIZE: usize = std::mem::size_of::<*const ()>();
```

Comment:
Slightly confusing variable name. `SIZEOF_POINTER` would be better. [Personally I'd just `use std::mem::size_of` and then always use `size_of::<*const ()>` in the code. I don't care too much though.]

Reply:
I would still need to have a constant as an `isize` tho unless I want to do `isize::try_from(size_of::<*const ()>).unwrap()` everywhere, which IMO distracts from the code quite a bit.

Reply:
Can I do two constants: `ISIZEOF_POINTER` and `USIZEOF_POINTER`?

Reply:
I prefer size_of, but it's something upon which reasonable people can differ. So long as we change the variable name to something less confusing, I'm OK with a variable.

Reply:
> Can I do two constants: ISIZEOF_POINTER and USIZEOF_POINTER?

If you need them, of course!

Reply:
I think I'd prefer two as having them as constants makes the main code much more readable. Renamed them in [5993877](https://github.com/ykjit/yk/pull/578/commits/599387756b3ae04da9bccab4d02c586b72e52a7a)

Reply:
Wrong commit?

Reply:
Aha it should be this one https://github.com/ykjit/yk/pull/578/commits/90ce72e5904bc878733751e61e90ac82f839d302

Reply:
Oh sorry.